pages/api/landingPage.test.js

```diff
@@ -0,0 +1,67 @@
+const axios = require('axios');
```

Comment:
Are we able to use the built in `fetch` library here instead of axios? Would like to avoid adding a new dependency since many are already out of date. https://developer.mozilla.org/en-US/docs/Web/API/fetch

Reply:
I will pass this unless it is HARD requirement.